test/test_nestedtensor.py

```diff
@@ -993,7 +993,7 @@ def test_matmul(self, device, dtype):
             r"matmul: Expected size for the 1st dimension of 2nd input tensor to be: [0-9]+ but got: [0-9]+.",
             lambda: torch.matmul(nt1, nt0)
         )
-        # error case: incompatible generalized batch size
+        # error case: incompatible (wrong) batch sizes that shouldn't even broadcast?
```

Comment:
Yes, this shouldn't work since matmul(torch.randn((2, 2, 4), torch.randn((3, 4, 6)) doesn't work.